lib/cfg-args.c

```diff
@@ -57,7 +57,7 @@ _resolve_unknown_blockargs_as_varargs(gpointer key, gpointer value, gpointer use
   CfgArgs *defaults = ((gpointer *) user_data)[0];
   GString *varargs = ((gpointer *) user_data)[1];
 
-  if (!defaults || cfg_args_get(defaults, key) == NULL)
+  if (!defaults || !cfg_args_contains(defaults, key))
```

Comment:
what about cfg_args_get() normalize_key() call? Isn't it missing here?

Reply:
Yeah, I also think `cfg_args_contains()` should normalize the key before the lookup.

Reply:
Nice catch! For some reason cfg_args_get() first tries to get without normalizing, and in case of failure, she falls back to normalizing. I don't know why, because cfg_args_set always normalizes. I skip the non-normalized check in my patch.